.github/workflows/wheels.yml

```diff
@@ -66,6 +44,16 @@ jobs:
         submodules: true
 
     - uses: pypa/cibuildwheel@v2.13.0
+      env:
+        # Target 64 bit architectures (x86_64, and arm64 on macOS)
+        CIBW_ARCHS: auto64
+        CIBW_ARCHS_MACOS: x86_64 arm64
+
+        # Temporary: use pre-release Python 3.12 for stable API builds
+        CIBW_PRERELEASE_PYTHONS: True
+
+        # GitHub Actions doesn't have macOS/arm64 runners, skip test on this platform
+        CIBW_TEST_SKIP "*-macosx_arm64"
```

Comment:
This just silences a warning. Since ARM runners are starting to show up (like on Cirrus CI), I thought it could be either removed or kept in the ci config.

Reply:
Ok, sure -- let's remove that line then.